installer/conf/sudoers

```diff
@@ -6,6 +6,10 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSRsyslog.post.sh
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSSysklog.post.sh
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSSyslog-ng.post.sh
 
+#Service status
+omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSServiceStat.sh
+omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSServiceStatAll.sh
```

Comment:
So, in last night's build, these scripts don't appear to exist:

```
jeffcof:~> ls /opt/microsoft/omsconfig/Scripts/
2.4x-2.5x/                          PerformInventory.py
2.6x-2.7x/                          PerformRequiredConfigurationChecks.py
3.x/                                protocol.py
client.py                           RegenerateInitFiles.py
GetDscConfiguration.py              RegisterHelper.sh
GetDscLocalConfigurationManager.py  Register.py
helperlib.py                        RemoveModule.py
ImportGPGKey.sh                     RestoreConfiguration.py
InstallModule.py                    SetDscLocalConfigurationManager.py
nxDSCLog.py                         StartDscConfiguration.py
OMS_MetaConfigHelper.py             StatusReport.sh
OMSRsyslog.post.sh                  TestDscConfiguration.py
OMSSysklog.post.sh                  zipfile2.6.py
OMSSyslog-ng.post.sh                zipfile2.6.pyc
jeffcof:~>
```
How, exactly, is this supposed to work?